### PR TITLE
Update SatoshiHero.fc

### PR DESCRIPTION
**Pull request by aviramdayan**
Update-Xpath-Elements-and-New-Way

### CODE REVIEW

File: Satoshi Hero/SatoshiHero.fc [hunk 1]
```diff
@@ -265,4 +330,38 @@ public class SatoshiHeroScript : FaucetScript
...
+        }
+		
+        var MinuteVAR = ElementByXPath("//h1[contains(@class,'error--text')]");
+        if (IsVisible(MinuteVAR))
+        {
+			Log("Check_Function - need to wait");
+			
+            var Minute2VAR = ElementByXPath("//h1[contains(@class,'error--text')]");
+			int MinINT, SecINT;
+            MinINT = Convert.ToInt32(Minute2VAR.Text.Trim().Split(':')[1])*60;
```
Comment (faucetcollector) [inline Satoshi Hero/SatoshiHero.fc]: better use int.TryParse

File: Satoshi Hero/SatoshiHero.fc [hunk 2]
```diff
@@ -83,166 +101,210 @@ public class SatoshiHeroScript : FaucetScript
...
+        if (IsVisible(MinuteVAR))
         {
-            return GetWaitSetting();
-        }
-        //let Faucet Collector continue.
+			Log("Check_Function - need to wait");
+			
+            var Minute2VAR = ElementByXPath("//h1[contains(@class,'error--text')]");
+			int MinINT, SecINT;
+            MinINT = Convert.ToInt32(Minute2VAR.Text.Trim().Split(':')[1])*60;
```
Comment (faucetcollector) [inline Satoshi Hero/SatoshiHero.fc]: Duplicate code and better use int.TryParse

File: Satoshi Hero/SatoshiHero.fc [hunk 3]
```diff
@@ -83,166 +101,210 @@ public class SatoshiHeroScript : FaucetScript
...
+		{	
+	
+          GoToUrl("https://satoshihero.com/en/login");
+          Wait();
+          Wait(3);
+		  
+          var EmailBox = ElementByXPath("//input[@name='login']");
+          if (!IsVisible(EmailBox)) 
+		  { 
+	      return Fail("Fail Setting email, Email box is not visible");
```
Comment (faucetcollector) [inline Satoshi Hero/SatoshiHero.fc]: 👍

Comment (aviramdayan) [reply Satoshi Hero/SatoshiHero.fc]: hehe :)

**Pull request merged**